tests/unit/s2n_extensions_server_key_share_select_test.c

```diff
@@ -423,12 +422,12 @@ int main()
             EXPECT_SUCCESS(s2n_extensions_server_key_share_select(server_conn));
 
             /* Server should update its choice to curve[0], no HRR */
```

Comment:
This comment is out of date. But you can probably just delete it.

Reply:
Fixed